ci/build.ps1

```diff
@@ -107,6 +107,9 @@ cmake -G $cmakeGenerator $(convertToCmakeArgs($nvimCmakeVars)) .. ; exitIfFailed
 cmake --build . --config $cmakeBuildType -- $cmakeGeneratorArgs ; exitIfFailed
 bin\nvim --version ; exitIfFailed
 
+# Ensure that the "win32" feature is set.
+bin\nvim -u NONE --headless -c 'exe !has("win32")."cq"' ; exitIfFailed
```

Comment:
Escape with double quotes with a backslash.

Reply:
https://github.com/janlazo/dotvim8/commit/a2855dc59e85a3c801a78776d534581924f8337f#diff-dad8a9269e73f782818bbba0843425f0R44

Reply:
```
bin\nvim -u NONE --headless -c 'exe !has(\"win32\").\"cq\"' ; exitIfFailed
```
?

Would the ampersand help here?  (but I think I've tried that in some variant)
`& bin\nvim -u NONE --headless -c "exe !has('win32').\"cq\"" ; exitIfFailed`

I assume you have Windows available at your hands?

Reply:
`.\bin\nvim.exe` is enough. powershell will autocomplete it like this even if you can run it with `./bin/nvim.exe`.
ampersand is useful for executing escaped paths.
`& "$NVIM_ABS_PATH" -u NONE --headless ...`

Reply:
> `.\bin\nvim.exe` is enough

What do you mean?  Just adding a dot in front here should fix it?

Reply:
Prepending with `.\` to run in the current directory. I got a warning in the default powershell terminal when I ran nvim without it.

Reply:
Ok.  But `bin\nvim --version` above does not hang.